node/task.ts

```diff
@@ -1197,9 +1191,9 @@ function legacyFindFiles_convertPatternToRegExp(pattern: string): RegExp {
     let flags = process.platform == 'win32' ? 'i' : '';
     return new RegExp(pattern, flags);
 }
-legacyFindFiles['legacyFindFiles_convertPatternToRegExp'] = legacyFindFiles_convertPatternToRegExp; // for unit testing
+_internal._legacyFindFiles_convertPatternToRegExp = _legacyFindFiles_convertPatternToRegExp; // for unit testing
```

Comment:
every internal member that needs to be exposed for the internal unit tests, now hangs off of `_internal`